requirements.txt

```diff
@@ -3,6 +3,8 @@ absl-py
 argparse
 cloud-tpu-diagnostics
 datetime
+google-cloud-compute==1.6.1
+google-cloud-monitoring==2.11.3
```

Comment:
No pin please! It is hard to maintain the pin -- who updates it and when?

Reply:
Done